sc/social/like/plugins/twitter/templates/plugin.pt

```diff
@@ -1,14 +1,7 @@
-<tal:twitter>
-    <!-- Twitter -->
-    <a href="https://twitter.com/share"
-       class="twitter-share-button"
-       tal:attributes="data-count view/typebutton;
-                       data-via   view/via;
-                       data-url   view/url;
-                       data-lang  view/language;
-                       data-text  here/Title;">Tweet</a>
-    <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document, 'script', 'twitter-wjs');</script>
-    <noscript>
-        <a tal:attributes="href view/urlnoscript">Tweet</a>
```

Comment:
I vote to keep this fallback when there is no javascript available (screen readers will thank you)

Reply:
that's no longer needed as we have no `<script>` tag now.

Reply:
I still see a `<script>` tag (but with **src** attribute, and this was a feature that will break after this merge.  People that could tweet with screen readers will not be able to do it anymore.